.github/workflows/release.yml

```diff
@@ -1,28 +1,31 @@
 name: "Release"
+
 on:
-  push:
-    tags:
-      - v**
+  workflow_call:
```

Comment:
`This used to trigger on tag push and release cachix-api/cachix, what's the process now? Is there something missing in `RELEASE.md`?

Reply:
The `workflow_call` lets it be triggered by another workflow. This happens after our CI runs successfully and only on version tags. https://github.com/cachix/cachix/pull/598/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R41-R45

It's not as elegant as having the version tag trigger right here, but at least our releases get run through CI before being published.